packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -66,18 +73,18 @@ const interpretationsWithIcons = [
   'low',
 ];
 
-const createEndIcons = (hasCommentIcon, hasModifiedIcon, hasUnverifiedIcon, resultKeyID) => {
+const createEndIcons = (hasCommentIcon, hasModifiedIcon, hasUnverifiedIcon, resultKeyID, intl) => {
   if (!hasCommentIcon && !hasModifiedIcon && !hasUnverifiedIcon) {
     return null;
   }
   let iconElements;
   if (hasUnverifiedIcon) {
-    iconElements = <IconUnverified className={cx('icon-unverified')} />;
+    iconElements = <IconUnverified className={cx('icon-unverified')} a11yLabel={intl.formatMessage({ id: 'Terra.clinicalResult.resultUnverified' })} role="img" focusable="true" />;
```

Comment:
The icon support for the FlowsheetResultCell is missing in this PR: https://github.com/cerner/terra-clinical/commit/e424c084c9b99423abe311ac078f612b9978051c
Added the icon support here. The corresponding tests are added as well.